arch/arm64/configs/pinenote_defconfig

```diff
@@ -1524,6 +1523,17 @@ CONFIG_CFG80211_DEBUGFS=y
 ##
 CONFIG_XFRM_USER=m
 
+##
+## file: security/Kconfig
+##
+CONFIG_LSM="landlock"
```

Comment:
What's the reason you choose ``landlock`` here? It's an improvement over the previous config, but by default it's a whole list, but you only enable "landlock" here.
I assume there's a good reason for it, but please document that in the commit message.

FWIW: The Debian kernel config has ``CONFIG_DEFAULT_SECURITY_APPARMOR=y``

Reply:
FTR: I would be absolutely fine to gradually enable these security features one-by-one and see what their effects are. Just document it in git commit message(s).